.template/addons/github/.github/workflows/test.yml.tt

```diff
@@ -144,7 +144,7 @@ jobs:
     needs: unit_tests
     runs-on: ubuntu-latest
     timeout-minutes: 5
-    if: github.event_name == 'pull_request'
+    if: github.event_name == 'pull_request' && !contains(join(github.event.pull_request.labels.*.name, ','), 'wip') && !contains(github.event.pull_request.title, '[wip]') && !github.event.pull_request.draft
```

Comment:
Also, could you check the case sensitivity here? For example, if we add `WIP` label in the PR instead of `wip`,  will it work?

Reply:
Yeah. As you can see [here](https://github.com/mosharaf13/fleet-searcher-ruby/pull/16), this check not case sensitive for labels. 

Refererence: https://docs.github.com/en/actions/learn-github-actions/expressions#contains

Reply:
Thanks for the confirmation! Good to know that they are not case-sensitive!